src/commands/showprofile.ts

```diff
@@ -0,0 +1,42 @@
+import { fetchUser } from "../util/UserUtil/index.js";
+import { EmbedBuilder } from "discord.js";
+
+const __showProfile: InteractionHandlerPayloads.GuildChatInputCommand = {
+    name: "showprofile",
```

Comment:
Convention for multi-word commands is commonly a dash.
Please change to `"show-profile"`.